### PR TITLE
[ISSUE #2666] Method appears to call the same method on the same object redundantly [LocalSubscribeEventProcessor]

### LINKED ISSUE #2666: [Enhancement] Method appears to call the same method on the same object redundantly [LocalSubscribeEventProcessor]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/LocalSubscribeEventProcessor.java line 83,99
analysis and explanation:
This method makes two consecutive calls to the same method, using the same constant parameters, on the same instance, without any intervening changes to the objects. If this method does not make changes to the object, which it appears it doesn't, then making two calls is just a waste

### Describe the solution you'd like

These method calls could be combined by assigning the result into a temporary variable, and using the variable the second time.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by chenyi19851209**
Fixes #2666 .

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by chenyi19851209** [comment]
fix for issue #2666

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/LocalUnSubscribeEventProcessor.java [hunk 1]
```diff
@@ -75,11 +75,11 @@ public void handler(final HandlerService.HandlerSpecific handlerSpecific, final
         final ChannelHandlerContext ctx = handlerSpecific.getCtx();
 
         final HttpEventWrapper requestWrapper = asyncContext.getRequest();
-
+        String localAddress = IPUtils.getLocalAddress();
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/LocalUnSubscribeEventProcessor.java:78]: When call IPUtils.getLocalAddress() one time,it's redundant code.

**Review by xwm1992 — DISMISSED** [review]

**Comment by xwm1992** [comment]
@chenyi19851209 please change the `LocalSubscribeEventProcessor` not `LocalUnSubscribeEventProcessor`

**Comment by chenyi19851209** [comment]
i have fix file LocalSubscribeEventProcessor for issue #2666

**Review by jonyangx — APPROVED** [review]

**Pull request merged**